tests/test_validate.py

```diff
@@ -1037,3 +1037,210 @@ def test_inject_ib_uuids_invalid(tmp_path, mocker):
     assert errors[0]["issue"] == validate_mbz_html.INVALID_IB_UUID_VIOLATION
 
     assert errors[0]["location"] == file_path
+
+
+def test_table_valid(tmp_path, mocker):
+    html = r"""
+<table class="os-raise-doubleheadertable">
+  <thead>
+    <tr>
+      <th scope="col"></th>
+      <th scope="col">\(x\)</th>
+      <th scope="col">\(+7\)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row">\(x\)</th>
+      <td>\(x^2\)</td>
+      <td>\(7x\)</td>
+    </tr>
+    <tr>
+      <th scope="row">\(+9\)</th>
+      <td>\(9x\)</td>
+      <td>\(63\)</td>
```

Comment:
Can we include an example of a `<td>` which includes a `<p class="something"></p>` somewhere for coverage on that scenario (want to make sure that doesn't flag an error)?